wagtailmenus/models.py

```diff
@@ -106,6 +106,12 @@ class MenuItem(models.Model):
         blank=True,
         help_text=_("Must be set if you wish to link to a custom URL."),
     )
+    link_handle = models.SlugField(
```

Comment:
I would rather use `CharField` here instead of `SlugField`. I know it's usage will likely be for things that fit a slug format, but it this is being added for developers to do what they like with, I feel we may as well leave it open to use as they like.